### PR TITLE
Support image push, in-app expiry, and push action buttons

### PR DESCRIPTION
**Pull request by ftcvlad**
### Description of Changes

Support for
1. message expiry
2. push images
3. push action buttons

For that update ObjSdk dependency, pass actionId through to JS

### Breaking Changes

-   None

### Release Checklist

Prepare:

-   [x] Detail any breaking changes. Breaking changes require a new major version number

Bump versions in:

-   [x] package.json
-   [x] plugin.xml
-   [x] src/ios/KumulosSDKPlugin.m
-   [x] src/android/.../KumulosInitProvider.java

Release:

-   [ ] Squash and merge to master
-   [ ] Delete branch once merged
-   [ ] Create tag from master matching chosen version
-   [ ] Fill out release notes
-   [ ] Run `npm publish`

### CODE REVIEW

File: src/android/PushReceiver.java [hunk 1]
```diff
@@ -26,6 +27,15 @@ static JSONObject pushMessageToJsonObject(PushMessage pushMessage) {
                 message.put("url", pushMessage.getUrl().toString());
             }
 
+            if (actionId != null){
+                 message.put("actionId", actionId);
+            }
+
+            String pictureUrl = pushMessage.getPictureUrl();
```
Comment (cgwyllie) [inline src/android/PushReceiver.java]: Think this return type is wrong? https://github.com/Kumulos/KumulosSdkAndroid/blob/master/kumulos/src/main/java/com/kumulos/android/PushMessage.java#L161

File: src/ios/KumulosSDKPlugin.m [hunk 2]
```diff
@@ -248,13 +248,16 @@ -(void)inAppPresentInboxMessage:(CDVInvokedUrlCommand*)command {
...
                            @"title": title,
                            @"message": message,
                            @"data": notification.data ?: NSNull.null,
                            @"url": url ?: NSNull.null
-                           };
+                           } mutableCopy];
 
+    if (notification.actionIdentifier){
+        [push setObject:notification.actionIdentifier forKey:@"actionId"];
+    }
```
Comment (cgwyllie) [inline src/ios/KumulosSDKPlugin.m]: Are we also going to add the picture URL here?

File: src/android/PushReceiver.java (hunk 1 above)
Comment (ftcvlad) [reply src/android/PushReceiver.java]: deleted the lines as dont pass in ObjSdk, so wouldn't be able to pass for ios platform without updating objSdk

File: src/ios/KumulosSDKPlugin.m (hunk 2 above)
Comment (ftcvlad) [reply src/ios/KumulosSDKPlugin.m]: rather remove from android

**Pull request merged**